### DISCUSSION TITLE
2.0版本怎么使用中国数据源，不科学上网直接获取不到数据

### DISCUSSION THREAD

**Discussion by cang-tong** [cangzhang/ChampR#326] [Q&A]
原来一直使用的1.3.11版本，AFK了一段时间后刚回归，发现原版本不弹一键符文的窗口了，使用2.0版本虽然能弹，但是需要科学上网，不是很方便。
说实话还是喜欢原来版本的界面，关于我这种情况 有什么好的建议吗

**Comment by cangzhang** [comment]
你的电信运营商是？
在我这边电信网络是可以正常访问的

**Reply by cang-tong** [reply]
我使用的是联通网络，网络类型应该是nat1,裸连的话就一直显示这个
![Uploading champr_rRxaDq1nB4.png…]()

**Comment by cangzhang** [comment]
现在域名绑定没有走 cf proxy，可以再试试能不能直连
